tests/test_cache_block_hashing.py

```diff
@@ -66,8 +66,8 @@ def test_auto_prefix_caching(model: str, block_size: int, max_num_seqs: int,
 
             hashes.append([])
             prompts = [prefix + prompt for prompt in sample_prompts]
-            seq_id = 0
-            for prompt in prompts:
+            # UPSTREAM SYNC: seq_id in enumerate needed to pass ruff
+            for seq_id, prompt in enumerate(prompts):
```

Comment:
is our ruff different? should we push this upstream?

Reply:
tests are not captured by upstream's ruff

Seems like they only run it on the vllm/vllm dir